src/sync.h

```diff
@@ -6,7 +6,10 @@
 #ifndef BITCOIN_SYNC_H
 #define BITCOIN_SYNC_H
 
+#include <logging.h>
```

Comment:
nit: If `PrintLockContention` stayed in the cpp file, this include can be avoided (it indirectly includes the heavy filesystem header).

Reply:
Good point, though we would have to essentially move `UniqueLock::Enter()` to the cpp file, due to calling `LogAcceptCategory(BCLog::LOCK)`. Punting on this for now.

Reply:
> nit: If `PrintLockContention` stayed in the cpp file, this include can be avoided (it indirectly includes the heavy filesystem header).

agree, dropped the commit changing this